docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -1,20 +1,22 @@
 # Calling a Phone Number
-Dialing a Phone Number allows you to send a phone number to dial on the user's phone. Regardless of platform (Android or iOS), you must be sure that a device is connected via Bluetooth (even if using iOS/USB) for this RPC to work. If it is not connected, you will receive a REJECTED `resultCode`.
+Dialing a Phone Number allows you to send a phone number to dial on the user's phone. Regardless of platform (Android or iOS), you must be sure that a device is connected via Bluetooth (even if using USB) for this RPC to work. If not connected to Bluetooth, you will receive a REJECTED @![iOS]`resultCode`!@ @![android,javaSE,javaEE] `Result`!@.
```

Comment:
Change: 
If not connected to Bluetooth, you will receive a REJECTED @![iOS]`resultCode`!@ @![android,javaSE,javaEE] `Result`!@.
to:
If the phone is not connected via Bluetooth, you will receive a result of `REJECTED` from Core.

Reply:
changed